packages/postgres-database/README.md

```diff
@@ -62,10 +62,6 @@ Upgrades to given revision (get ``info`` to check history)
 
 ## Database Models
 
-Entity Relationship Diagram (ERD) defined under ``simcore_postgres_database.models``:
-
-![scripts/create_erd.py-output](doc/img/postgres-database-models.svg)
```

Comment:
is this script deprecated?

Reply:
The idea is to remove all the svgs since either they are not up-to-date or nobody is using them
Instead, i will add a container with graphviz preinstalled and all the tools to create graphs from code 
This way we can use it as an analysis tool